u2f-ref-code/java/tests/com/google/u2f/server/ControlFlagsTest.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * 
+ */
+package com.google.u2f.server;
+
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @author at
+ *
+ */
+public class ControlFlagsTest {
+  protected byte userPresent, transferAccessMessage, both, neither;
+
+  @Before
+  public void setUp() throws Exception {
+    userPresent = 0x1;
+    transferAccessMessage = 0x2;
+    both = 0x3;
```

Comment:
you don't need to re-initialize them every time.

Just make them private static final int